docker-for-windows/install-windows-home.md

```diff
@@ -0,0 +1,146 @@
+---
+description: How to install Docker Desktop on Windows 10 Home
+keywords: Windows 10 Home, Home, Windows, install, download, run, Docker, local
+title: Install Docker Desktop on Windows Home
+---
+
+You can now install Docker Desktop on Windows 10 Home machines. Docker Desktop for WSL 2 Windows Home is a full version of Docker Desktop for Linux container development.
+
+This page contains information on installing Docker Desktop on Windows 10 Home. If you are looking for information about installing Docker Desktop on Windows 10 Pro, Enterprise, or Education, see [Install Docker Desktop on Windows](/install.md).
+
+[Download from Docker Hub](Add URL){:
+.button .outline-btn}
+
+Docker Desktop for WSL 2 Windows Home offers the following benefits:
+
+- Latest version of Docker on your Windows Home machine
+- Install Kubernetes in one click on Windows Home
+- Integrated UI to view and manage your running containers
+- Start Docker Desktop in less than ten seconds
+- Use Linux Workspaces
+- Dynamic resource and memory allocation
+- Networking stack, support for http proxy settings, and trusted CA synchronization
+
+For detailed information about WSL 2, see [Docker Desktop WSL 2 backend](/wsl-tech-preview.md)
+
+## What to know before you install
+
+### System Requirements
+
+Windows 10 Home machines must meet the following requirements to install Docker Desktop:
+
+  - Install Windows 10 Insider Preview build 19041 or higher.
+  - The following hardware prerequisites are required to successfully run Client
+Hyper-V on Windows 10:
+
+     - 64 bit processor with [Second Level Address Translation (SLAT)](http://en.wikipedia.org/wiki/Second_Level_Address_Translation)
+     - 4GB system RAM
+    - BIOS-level hardware virtualization support must be enabled in the
+    BIOS settings.  For more information, see
+    [Virtualization](troubleshoot.md#virtualization-must-be-enabled).
+  - Enable WSL 2 feature on Windows. For detailed instructions, refer to the[Microsoft documentation](https://docs.microsoft.com/windows/wsl/wsl2-install).
+  - Download and install the [Linux kernel update package](https://docs.microsoft.com/windows/wsl/wsl2-kernel).
+
+> **Note:** Docker supports Docker Desktop on Windows based on Microsoft’s support lifecycle for Windows 10 operating system. For more information, see the [Windows lifecycle fact sheet](https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet).
+
+### What's included in the installer
+
+The Docker Desktop installation includes [Docker Engine](/engine/index.md),
+Docker CLI client, [Docker Compose](/compose/index.md),
+[Notary](/notary/getting_started.md),
+[Kubernetes](https://github.com/kubernetes/kubernetes/),
+and [Credential Helper](https://github.com/docker/docker-credential-helpers/).
+
+Containers and images created with Docker Desktop are shared between all
+user accounts on machines where it is installed. This is because all Windows
```

Comment:
This is not true on Windows Home

Reply:
It is not the case with WSL2 on pro either

Reply:
Removed it from the Windows Home topic. Added a caveat in the Windows Pro topic that it is not possible to share containers and images when using WSL 2.

Reply:
Fixed.